content/about/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+
+## The European Commission and the Digital Markets Act
+
+To prevent extreme concentration of power in the messaging sector, the **European Commission introduced the Digital Markets Act (DMA)**. Its goal is to ensure that communication remains **open, fair, and competitive**, preventing a future where global messaging is controlled by a **monopoly, duopoly, or, at best, an oligopoly**. Enforcing interoperability is the only way to allow users to **freely choose their preferred messaging provider** rather than being locked into dominant platforms.
+
+In this spirit—enabling a truly **competitive market** where users can select messaging providers that align with their **budget, business model, and privacy preferences**—we must return to the **core principle of interoperability: the adoption of open communication standards**.
+
+This is the intent behind the DMA’s requirements.
+
+## Why the XSF is Concerned About Meta’s Proposal and Why XMPP is the Solution
+
+Now that Meta has submitted its proposal for messaging interoperability to the European Commission, we have serious concerns that their approach could not only **preserve the status quo** but also **give Meta even more control over user data**.
+
+### Key Concerns with Meta’s Interoperability Proposal
+
+#### 1. Restrictive Access Through NDAs and Agreements
+
+Meta requires companies to **sign a non-disclosure agreement (NDA)** to access their interoperability detailed documents and likely demands an agreement before enabling interoperability between networks. This effectively **keeps companies in a gatekeeper relationship**, just as Meta did when launching *Messenger for Businesses* to control interactions and ensure benefits flowed primarily to them. 
+
+For competitors, **signing an agreement with Meta grants them disproportionate leverage** due to their massive user base. This is **not true interoperability**—it is a **negotiated privilege**, reinforcing Meta’s dominant position.
+
+Requiring an NDA **prevents non-profit organizations like the XSF** from evaluating or contributing to the interoperability mechanism. It **blocks open-source developers from implementing interoperability** in their software, severely limiting innovation.
+
+#### 2. API-Based Approach Instead of Open Protocols
+
+Meta’s proposal appears to be based on **APIs rather than open protocols**. This is a fundamental problem because:
+
+- **Protocols** are designed for **interop and cooperation**, ensuring long-term stability.
+- **APIs** are controlled by the provider and offer a **limited set of features**, with associated **rate limits, fees, and restrictions** that can change over time.
```

Comment:
Rate limits are a fundamental abuse prevention technique at all levels of the stack. Proposing "usage limits" for clarity in the context of APIs.

```suggestion
- **APIs** are controlled by the provider and offer a **limited set of features**, with associated **usage limits, fees, and restrictions** that can change over time.
```